### PR TITLE
Add dummy muon momentum scale, more datasets, trigger

### PR DESCRIPTION
**Pull request by kdlong**

### CODE REVIEW

File: scripts/histmakers/mw_with_mu_eta_pt.py [hunk 1]
```diff
@@ -20,19 +19,17 @@
...
+datasets = wremnants.datasets2016.getDatasets(maxFiles=args.maxFiles)
 
 era = "GToH"
 
 muon_prefiring_helper, muon_prefiring_helper_stat, muon_prefiring_helper_syst = wremnants.make_muon_prefiring_helpers(era = era)
 
-wprocs = ["WplusmunuPostVFP", "WminusmunuPostVFP"]
-zprocs = ["ZmumuPostVFP"]
+wprocs = ["WplusmunuPostVFP", "WminusmunuPostVFP", "WminustaunuPostVFP", "WplustaunuPostVFP"]
+zprocs = ["ZmumuPostVFP", "ZmumuPostVFP"]
```
Comment (bendavid) [inline scripts/histmakers/mw_with_mu_eta_pt.py:32]: Second one should be Ztautau?

File: wremnants/include/muonCorr.h [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+double scaleWeight(double weight, double scale) {
+    return std::exp(scale*std::log(std::abs(weight))*std::copysign(1., weight));
+}
+
+template <size_t ETABINS, size_t T>
+Eigen::TensorFixedSize<double, Eigen::Sizes<2, ETABINS>> dummyScaleFromMassWeights(
+    Eigen::TensorFixedSize<double, Eigen::Sizes<T>>& weights, double eta, double scale, bool isW=true) {
+    const double refMass = isW ? 80351.81229 : 91153.50974;
+    const size_t centralIdx = 10;
```
Comment (bendavid) [inline wremnants/include/muonCorr.h:12]: Central index is the same for both W and Z?

File: wremnants/include/muonCorr.h [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    const double scaleMeV = refMass*scale;
+    const int step10MeV = static_cast<int>(scaleMeV/10)+1;
```
Comment (bendavid) [inline wremnants/include/muonCorr.h:14]: this kind of stuff is probably clearer if std::floor/ceil is used explicitly

File: wremnants/datasets/datasets2016.py [hunk 4]
```diff
@@ -8,55 +8,99 @@
...
+        xsec = 1976.1,
+    )
+
+    BR_TAUToMU = 0.17394
+    ZttPostVFP = narf.Dataset(name = "ZtautauPostVFP",
+        filepaths = makeFilelist(
+            ["/scratch/shared/originalNANO/DYJetsToTauTau_postVFP/*/*.root"], maxFiles),
+        is_data = False,
+        # At least one tau->e or mu decay, so everything that's not all other decays
+        xsec = ZmmPostVFP.xsec*(1.-(1.-(BR_TAUToMU+0.1782)**2)),
```
Comment (bendavid) [inline wremnants/datasets/datasets2016.py:41]: 0.1782 is BR_TAUToE?  (make it a variable like BR_TAUToMU?)

File: wremnants/include/muonCorr.h (hunk 2 above)
Comment (kdlong) [reply wremnants/include/muonCorr.h:12]: Yes it is, the two extra in Z are the width variations (or sinthetaW, can't remember off the top of my head).

File: wremnants/include/muonCorr.h (hunk 3 above)
Comment (kdlong) [reply wremnants/include/muonCorr.h:14]: You mean instead of just integer division?

File: wremnants/datasets/datasets2016.py (hunk 4 above)
Comment (kdlong) [reply wremnants/datasets/datasets2016.py:41]: ya ok

**Pull request merged**